lib/px/block/pxtemplate.lua

```diff
@@ -21,7 +21,8 @@ function M.load(config_file)
         local js_client_src = string.format('//client.perimeterx.net/%s/main.min.js', px_config.px_appId)
         local collectorUrl = 'https://collector-' .. string.lower(px_config.px_appId) .. '.perimeterx.net'
         local captcha_url_prefix = 'https://' .. px_config.captcha_script_host
-        if px_config.first_party_enabled then
+        -- in case we are in first party mode (not relevant for mobile), change the base paths to use first party
+        if px_config.first_party_enabled and not string.match(template, "mobile")then
```

Comment:
we can use here `ngx.ctx.px_cookie_origin == header` instead of matching the mobile?

Reply:
yes, I can